app/components/article-content.js

```diff
@@ -300,6 +307,49 @@ export default Component.extend(
 			});
 		},
 
+		/**
+		 * FIXME FEATURED VIDEO A/B TEST ONLY
+		 */
+		renderFeaturedVideo() {
+			const $infoboxes = this.$('.portable-infobox'),
+				$placeholder = $('<div />');
+
+			if ($infoboxes.length) {
+				$infoboxes.first().after($placeholder);
+			} else {
+				this.$().children('h2').first().after($placeholder);
+			}
+
+			this.renderedComponents.push(
+				this.renderComponent({
+					name: 'article-featured-video',
+					attrs: {
+						model: this.get('featuredVideo')
+					},
+					element: $placeholder.get(0)
+				})
+			);
+
+			this.adjustVideoScale();
+		},
+
+		/**
+		 * To not mess with side paddings across whole article.
+		 */
```

Comment:
let's talk about it offline ;)

Reply:
this will be probably removed anyway as it seems like scaling breaks captions' position